demo/tpu-training/inception-v3-tpu.yaml

```diff
@@ -64,7 +64,7 @@ spec:
           # [REQUIRED] Must specify the Google Cloud Storage location where your
           # output model will be stored.
         - name: MODEL_BUCKET
-          value: "gs://<my-model-bucket>/inception_v3"
+          value: "gs://vishh/tf-models/inception_v3"
```

Comment:
OK to leave your bucket here?

Reply:
Yeah. It will be easier for the demo. I'll clean it up later.